src/main/resources/application.yaml

```diff
@@ -97,7 +97,7 @@ idam:
     totp_secret: ${S2S_KEY:AAAAAAAAAAAAAAAA}
     microservice: em_npa_app
   s2s-authorised:
-    services: ${S2S_NAMES_WHITELIST:sscs,divorce,ccd,em_gw,jui_webapp,pui_webapp,xui_webapp}
+    services: ${S2S_NAMES_WHITELIST:ccd,em_gw,xui_webapp}
```

Comment:
https://github.com/hmcts/cnp-flux-config/pull/23023#discussion_r1202065198

Reply:
Please refer to comment on other PR

Reply:
I have checked AppInsights logs and can see only Xui_WebApp is the service calling Redaction on AAT & Prod. So don't see the need to change this in lower env. first and then do the change for rest of it.